.github/workflows/build-back-end-rdevelopment.yml

```diff
@@ -0,0 +1,26 @@
+name: deploy-development-back-end-container
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - develop
+    paths:
+      - "backend_server/**"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged
+
+    steps:
+      - name: ACR build back-end
+        id: acr-back-end
+        uses: ams0/acr-task-github-action@master
+        with:
+          service_principal: ${{ secrets.service_principal }}
+          service_principal_password: ${{ secrets.service_principal_password }}
+          tenant: ${{ secrets.tenant }}
+          registry: ${{ secrets.registry }}
+          repository: ${{ secrets.repository }}
+          image: back-end
+          folder: backend_server
+          tag: development
```

Comment:
We are using the shorter word "develop" in quite many places, would it make more sense to use it here instead of development as well?

Reply:
If it feels more informative I'm happy to go with "development" as well!